googleCalendar@javahelps.com/files/googleCalendar@javahelps.com/lib/xdate.js

```diff
@@ -98,7 +98,7 @@ var XDate = function(t, e, n, r) {
 
     function T(e, n, r) {
         for (var i, o = u.parsers, s = 0; s < o.length; s++)
-            if (i = o[s](e, n, r)) return i;
+            if (i == o[s](e, n, r)) return i;
```

Comment:
Hi @claudiux 
With this change, my calendar events are shown one day earlier. My program on Monday is being displayed as on Sunday. Once I changed this line back to `if (i = o[s](e, n, r)) return i;` it works fine.

Reply:
Just noticed that there is a related issue https://github.com/linuxmint/cinnamon-spices-desklets/issues/459

Reply:
Hi @slgobinath,

Please try to replace the function T by this one:
```
function T(e, n, r) {
        for (var i, o = u.parsers, s = 0; s < o.length; s++) {
            i = o[s](e, n, r);
            if (i) return i;
        }
        return r[0] = new t(e), r
    }
```

Reply:
Hi @claudiux 
That works. I already sent a PR https://github.com/linuxmint/cinnamon-spices-desklets/pull/460